packages/analytics/test/analytics-data.spec.ts

```diff
@@ -1,3 +1,4 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
```

Comment:
I guess this is generated from the eslint, or the eslint shows error/warning?

Reply:
Yes, I condensed it to one line, because it was easier to implement and actually nicer to loose only one line of code on that.